api/src/main/java/com/epam/pipeline/manager/security/GrantPermissionManager.java

```diff
@@ -382,6 +385,7 @@ public void filterTree(AclSid aclSid, AbstractHierarchicalEntity entity, Permiss
         if (aclSid.isPrincipal()) {
             filterTree(aclSid.getName(), entity, permission);
         } else {
+            roleManager.loadRole(aclSid.getName());
```

Comment:
Result of the method call is ignored, is it ok? I assume non-principal sid may be a `group` not only `role`.